lib/web_ui/lib/src/engine/bitmap_canvas.dart

```diff
@@ -93,6 +93,11 @@ class BitmapCanvas extends EngineCanvas {
     _childOverdraw = value;
   }
 
+  // Maps src url to image element to reuse image instances across frames.
+  // Optimization for some browsers that keep refreshing images causing
+  // flicker.
+  Map<String, List<html.ImageElement>> _reusableImages;
```

Comment:
Can we make this global and clean up in `commitScene` instead of `endOfPaint`? This way images can be reused across canvases (we frequently create a new canvas because of a failed match, but it will contain the same image from the previous frame), we will allocate fewer maps and lists, and we can remove several null checks (we can keep the global map always non-null).

Reply:
Moved cache to picture level, so it survives across picture updates.